app/Controllers/UserLogin.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Controllers;
+
+use CodeIgniter\API\ResponseTrait;
+use App\Models\UserModel;
+
+class UserLogin extends BaseController
+{
+    use ResponseTrait;
+
+    public function index()
+    {
+        if($this->isLoggedIn) {
+            return redirect()->to("/home");
+        } else {
+            return view('pages/visitor_home');
+        }
+    }
+
+    public function login()
+    {
+        $request = \Config\Services::request();
+        $data    = $request->getPost();
+
```

Comment:
$email = $data['email'];
password is same on below.

Reply:
done